store/cachekv/internal/memiterator.go

```diff
@@ -8,12 +8,12 @@ import (
 	"github.com/tidwall/btree"
 )
 
-var _ types.Iterator = (*memIterator)(nil)
+var _ types.Iterator = (*MemIterator)(nil)
 
 // memIterator iterates over iterKVCache items.
 // if key is nil, means it was deleted.
 // Implements Iterator.
-type memIterator struct {
+type MemIterator struct {
```

Comment:
given that all fields are private, I don't understand why making this public.

Reply:
Ok